RecoHGCal/TICL/plugins/HGCGraph.cc

```diff
@@ -53,6 +56,9 @@ void HGCGraph::makeAndConnectDoublets(const TICLLayerTiles &histo,
                     if (mask[innerClusterId] == 0.)
                       continue;
                     auto doubletId = allDoublets_.size();
+                    if (maxDeltaTime != -1 &&
```

Comment:
Isn't it too deep to define a magic number?
Perhaps use ```<= 0``` to be less restrictive

Reply:
-1 was meant to be "don't check the time compatibility", to leave the option to switch this on or off and instead of adding a bool I used a negative value.
I can add a comment here, or add a bool instead.